rs/nns/governance/tests/governance.rs

```diff
@@ -26,7 +26,7 @@ use ic_nervous_system_common::{
     cmc::CMC,
     ledger,
     ledger::{compute_neuron_staking_subaccount_bytes, IcpLedger},
-    NervousSystemError, E8, ONE_DAY_SECONDS, ONE_YEAR_SECONDS,
+    NervousSystemError, E8, ONE_DAY_SECONDS, ONE_MONTH_SECONDS, ONE_YEAR_SECONDS,
```

Comment:
NCR;

I don't like that we have both these constants at the same time. Why can't we use just one of them?

Is `ONE_MONTH_SECONDS * 12 == ONE_YEAR_SECONDS`?

Reply:
> Is `ONE_MONTH_SECONDS * 12 == ONE_YEAR_SECONDS`?

Yes. This is an abomination. However, this is already a public aspect of the NNS -> it makes sense for us to have a constant.

We certainly could have just ONE_YEAR_SECONDS, but integer division is kind of a pain (because you then need to think about whether there is any remainder). It just so happens that ONE_YEAR_SECONDS **_IS_** in fact divisible by 12, but this is non-obvious to most people, esp if you tell them one year is nominally defined as EXACTLY 365.25 days.

Reply:
Okay, so if the equality holds, that's good.

Maybe we should add a test to keep it that way? 

NCR, as this PR didn't introduce those constants.

Reply:
Ok. Let me do that in another PR.